omnibus/files/private-chef-ctl-commands/chef-server-ctl

```diff
@@ -0,0 +1,328 @@
+#!/opt/opscode/embedded/bin/ruby
+
+require 'rubygems'
+# Because we're not installed as a gem ourselves,
+# we'll need to explicitly load the omnibus-ctl gem
+# so that we can see it
+# so that we can see it.
+gem 'omnibus-ctl'
+require "omnibus-ctl"
+
+module Omnibus
+
+  # We keep the base behavior of Omnibus::Ctl, but
+  # add our own hooks for before/after command run.
+  # In particular we hook the service calls.
+  class ChefServerCtl < Omnibus::Ctl
+    def command_post_hook(*args)
```

Comment:
Why isn't this the default implementation of post_host?  It seems like this is the behavior we might want generally?